buildSrc/src/main/kotlin/Config.kt

```diff
@@ -25,19 +25,16 @@ object Config {
         }
 
         object Support {
```

Comment:
Rename to `Jetpack`?

Reply:
Renamed to `Androidx` since `Arch` is part of Jetpack too but we kept that separate from support before.